interview-questions.md

```diff
@@ -14,44 +14,62 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is how we send and receive changes to data from our local to another local or to a database. Github is a graphic user interface for git that streamlines the changes to data by letting users review pull requests and merge them afterwards.
 
-Researched answer:
+Researched answer: Git (Global Information Tracker) is a version control system that allows for many people to contribute and share changes in code to the same code base. Github is a GUI that allows users to manage git repositories and offers more funtionality and resources when pushing and pulling code branches to and from git repositories.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Comparison operators (==, ===, !==, etc) are used to compare two values and returns a Boolean value (true/false)
 
-Researched answer:
+Researched answer: Comparison operators, logical operators (&&, ||), relational operators (in, instanceof)
```

Comment:
Good job researching and listing more operators that will return a Boolean value.